### ISSUE TITLE
Autotype fails

### ISSUE THREAD

**Issue by Battleman** [keeweb/keeweb#1205]
**Describe the bug**
When using autotype on linux, it fails after first password character (username works just fine). Happens with and without obfuscation, normal or global autotype.

**To Reproduce**
1. Open any program (firefox, chrome, even text editor)
2. Select entry from keeweb
3. perform autotype (or global autotype)

**Expected behavior**
Autotype sequence should complete (username, tab, password, enter)

**Screenshots**
Username works fine, but only first pwd character is displayed ![Result after autotype](https://user-images.githubusercontent.com/6746316/60007578-db3a9b00-9672-11e9-9db6-9997fbee8a30.png)
After first character, it stops and this happens in keeweb:
![Autotype error](https://user-images.githubusercontent.com/6746316/60007653-06bd8580-9673-11e9-84d9-589d781049ec.png)

**Environment**
```KeeWeb v1.8.2 (ebe7d35c, 2019-04-22)
Environment: electron v4.0.1
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.8.2 Chrome/69.0.3497.106 Electron/4.0.1 Safari/537.36
```
**Kdbx File**
 It happens with a new database. Attached here.
[testdb.kdbx.zip](https://github.com/keeweb/keeweb/files/3320142/testdb.kdbx.zip)

**Logs**
```
[DEBUG] 2019-06-24T09:16:30.751Z [auto-type] Auto type event {"id":"238f3588-f360-1afb-2aeb-24562b3ebc9a:P7ntnW8k5UiWbH1WkNjI4A==","uuid":"P7ntnW8k5UiWbH1WkNjI4A=="}
[DEBUG] 2019-06-24T09:16:30.751Z [auto-type] Hide window
[DEBUG] 2019-06-24T09:16:30.875Z [auto-type] Start {USERNAME}{TAB}{PASSWORD}{ENTER}
[DEBUG] 2019-06-24T09:16:30.875Z [auto-type] Parsed [op:USERNAME,op:TAB,op:PASSWORD,op:ENTER]
[DEBUG] 2019-06-24T09:16:30.876Z [auto-type] Resolved [******************************,key:tab,[*,*,*,*,*,*,*,*,*,*,*,*,*,*],key:enter]
[INFO ] 2019-06-24T09:16:31.429Z [launcher] spawn xdotool: 0, 554ms
[ERROR] 2019-06-24T09:16:31.559Z [launcher] spawn xdotool: 2, 128ms

Error: Invalid key sequence ']'
Failure converting key sequence ']' to keycodes
Error: Invalid key sequence ']'
Failure converting key sequence ']' to keycodes
xdo_send_keysequence_window reported an error for string ']'
[ERROR] 2019-06-24T09:16:31.559Z [auto-type] Run error Exit code 2
```
The odd symbols, characterized as invalid sequence, are this (they don't like copy-paste):
![Screenshot from 2019-06-24 11-22-39](https://user-images.githubusercontent.com/6746316/60007375-797a3100-9672-11e9-93d5-7030e88fe121.png)

I hope everything is here.

**Comment by antelle** [comment]
Hi! Are there any special characters in the password?

**Comment by Battleman** [comment]
No, only alphanumerical. The test database attached has a % sign in the password, but all my other test had normal chars.

**Comment by antelle** [comment]
The bot was misconfigured, sorry for a wrong notification.

**Comment by Battleman** [comment]
Apparently, fixed in v1.10.0
